kernel/solo5.h

```diff
@@ -110,7 +110,7 @@ int solo5_console_write(const char *buf, size_t n);
 /*
  * Exits the application.
  */
-void solo5_exit(void) __attribute__((noreturn));
+void solo5_exit(int exit_status) __attribute__((noreturn));
```

Comment:
Use `status` instead of `exit_status`.

Do we want to follow the UNIX `exit()` and type `status` as `int`, or not? I'm happy keeping `int` since it's easier to translate back to an exit status of the `ukvm` process, but just checking for opinions.

Reply:
Changed.

The main reason i can think of is the one you mention: map unikernel errors with ukvm. Would the alternative be something more than an int? something that includes a reason for failure (i.e., a string)?

Reply:
> The main reason i can think of is the one you mention: map unikernel errors with ukvm. Would the alternative be something more than an int? something that includes a reason for failure (i.e., a string)?

Eh, forget it. Not worth it. And returning strings in a C-style ABI is horrible anyway.